web/views.py

```
@@ -137,6 +138,7 @@
 from .referrals import send_referral_reward_email
 from .social import get_social_stats
 from .utils import get_or_create_cart
+from .social_share import create_social_share_discount, notify_user_about_discount, apply_social_share_discount, verify_social_share
```

Ruff (0.8.2)

`.social_share.create_social_share_discount` imported but unused

Remove unused import

(F401)